src/lib/rest/rest.cpp

```diff
@@ -1298,7 +1298,7 @@ static int connectionTreat
     //
     // Transaction starts here
     //
-    lmTransactionStart("from", ip, port, url);  // Incoming REST request starts
+    lmTransactionStart("from", "", ip, port, url);  // Incoming REST request starts
```

Comment:
We have ip, port and url, but not scheme ... ? A little strange, scheme should have its place also, now that we care about it ...    Right?  It is part of ip?  A simple strchr could parse that and separate the two ...